elm.json

```diff
@@ -16,9 +16,11 @@
             "mdgriffith/elm-ui": "1.1.5"
         },
         "indirect": {
+            "Skinney/murmur3": "2.0.8",
             "elm/bytes": "1.0.8",
             "elm/file": "1.0.5",
             "elm/json": "1.1.3",
+            "elm/regex": "1.0.0",
```

Comment:
Why were these added?
Indirect dependencies are automatically added when a new direct dependency is added. There are no new direct dependencies, so these are not needed.